recipes/grpc/all/conanfile.py

```diff
@@ -60,6 +60,13 @@ def requirements(self):
         self.requires('abseil/20210324.1')
         self.requires('re2/20210401')
 
+    def build_requirements(self):
+        self.build_requires('protobuf/3.17.1')
+
+        #when cross compiling we need pre compiled grpc plugins for protoc
+        if tools.cross_building(self):
```

Comment:
Use:
```suggestion
        if hasattr("settings_build") and tools.cross_building(self):
```

you only need to build-requires `gRPC` if it is an actual cross-building scenario.

Reply:
Just arrived a commit with the same idea 😅

Reply:
nice thanks